components/search_results/index.jsx

```diff
@@ -27,25 +27,38 @@ function makeMapStateToProps() {
     let isFlaggedByPostId;
 
     return function mapStateToProps(state) {
+        const config = getConfig(state);
+
+        const dataRetentionEnableMessageDeletion = config.DataRetentionEnableMessageDeletion === 'true';
+        const dataRetentionMessageRetentionDays = config.DataRetentionMessageRetentionDays;
```

Comment:
I know other numeric fields in the config require a `parseInt` here -- can you confirm this is not required?

Reply:
I simply move it from the bottom of the function to here, but I can check it and fix it if needed here :)

Reply:
It is not required because is used as value of a FormattedText, so, it will be converted to string anyway.